### PR TITLE
Wd 6605 verify credly works and replace the api keys and org id in kubernetes secrets

### PR DESCRIPTION
**Pull request by abhigyanghosh30**
## Done

- Change template IDs for the badges in the API 
- Removed visibility for webhook responses page for non-staff

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
### Proper QA
- Take the sample exam on TA
- Pass the said exam
- After a few hours, a badge should appear in [/credentials/your-badges](https://ubuntu-com-13218.demos.haus/credentials/your-badges) page
### Easier QA
- Visit https://app.trueability.com/m/ability_screens/4190/webhooks/202/test
- Response should be {"status":"badge_not_issued"}, 403
- This means that the website is accepting the webhooks
- Now call [this endpoint](https://ubuntu-com-13218.demos.haus/credentials/assessment_passed) using cURL/Postman.
{ "assessment":{ "score":0.75, "user":{ "full_name":"<YOUR_NAME>", "email":"<YOUR_EMAIL>" }, "ability_screen":{ "cutoff_score": 0.6 }, "ability_screen_variant":{ "ability_screen_id":4190 } } }
A badge should be issued from Credly
Verify new badge in [/your-badges](https://ubuntu-com-13218.demos.haus/credentials/your-badges)

## Issue / Card

Fixes [#WD-6605](https://warthogs.atlassian.net/browse/WD-6605)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -7,7 +7,7 @@ STRIPE_PUBLISHABLE_KEY=pk_test_yndN9H0GcJffPe0W58Nm64cM00riYG4N46
 STORE_MAINTENANCE=false
 BADGR_URL=https://api.test.badgr.com
 TRUEABILITY_URL="https://app.trueability.com"
-CREDLY_URL="https://sandbox-api.credly.com/v1"
```
Comment (albertkol) [inline .env:10]: Is working with Prod API a good idea on local? 
If not, leave the sandbox url here.

File: konf/site.yaml [hunk 2]
```diff
@@ -603,7 +603,7 @@ staging:
         name: trueability
 
     - name: CREDLY_URL
-      value: https://sandbox-api.credly.com/v1
```
Comment (albertkol) [inline konf/site.yaml:606]: Same question. Is using prod api link good for staging? If not, change back.

File: webapp/shop/api/credly/api.py [hunk 3]
```diff
@@ -17,9 +17,9 @@ def __init__(
         self.session = session
         self.auth_token = auth_token
         self.badge_template_dict = {
-            4190: "3d3a19a1-324c-48fa-aa55-60f66c0dcca6",
```
Comment (albertkol) [inline webapp/shop/api/credly/api.py:20]: I assume here you are changing the keys to match prod keys.
Are you sure you want to use prod API on all environments? If no, these values should be in the env file and konf.

**Review by albertkol — COMMENTED** [review]
One question asked about 3 time.

File: .env (hunk 1 above)
Comment (abhigyanghosh30) [reply .env:10]: Sandbox is not active anymore. Everything will have to be done on prod. So how we tackle that is for staging we have mapped to exam 4190 which is a staging exam and that gives out a staging badge. The production exam will give out a production badge

**Pull request merged**